recipes/ssh-crypt/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "ssh-crypt" %}
+{% set version = "1.1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ssh-crypt-{{ version }}.tar.gz
+  sha256: 69cd2e52dc33c6e92b16247ca50a66325b868899125a3d9a0c4c7b88e9283a33
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

This is pure Python and could be noarch:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
  noarch: python
```

---

The pipeline fails under win64:
> [noarch_python] Noarch package contains binary script: ssh-crypt.exe

[log](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=631529&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=2fb5b0a7-737c-5f5c-8f3f-f6db6174bacf)

---

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points